grpc/python/BUILD

```diff
@@ -78,7 +80,7 @@ assemble_pip(
     author = "Grakn Labs",
     author_email = "grabl@grakn.ai",
     license = "AGPLv3+",
-    install_requires = [],
+    install_requires_file = "//grpc/python:requirements.txt",
```

Comment:
shouldn't this be `requirements_file` @vmax ? I remember us having a discussion about that